### PR TITLE
Add read xml function

### PR DESCRIPTION
**Pull request by RachelNickerson**
Added new function to read in xml files

### CODE REVIEW

File: catchment/models.py [hunk 1]
```diff
@@ -34,6 +34,28 @@ def read_variable_from_csv(filename):
 
     return newdataset
 
+def read_variable_from_xml(filename):
+    """Reads a named variable from a XML file, and returns a
+    pandas dataframe containing that variable.
+
+    :param filename: Filename of CSV to load
+    :return: 2D array of given variable. Index will be dates,
```
Comment (jurezgubic) [inline catchment/models.py:42]: Might be good to make this more readable? Maybe write it in the form of a poem?

**Review by jurezgubic — CHANGES_REQUESTED** [review]
Lovely feature. I also enjoy xml.

File: catchment/models.py (hunk 1 above)
Comment (RachelNickerson) [reply catchment/models.py:42]: Limerick, haiku, iambic pentameter?

Comment (jurezgubic) [reply catchment/models.py:42]: haiku!

**Pull request closed without merging**